source/postprocess/particles.cc

```diff
@@ -100,18 +100,12 @@ namespace aspect
             // If the property has dim components, we treat it as vector
             if (n_components == dim)
               {
-#if DEAL_II_VERSION_GTE(9,1,0)
                 vector_datasets.push_back(std::make_tuple(property_index_to_output_index[field_position],
                                                           property_index_to_output_index[field_position]+n_components-1,
                                                           field_name,
                                                           DataComponentInterpretation::component_is_part_of_vector));
-#else
-                vector_datasets.push_back(std::make_tuple(property_index_to_output_index[field_position],
-                                                          property_index_to_output_index[field_position]+n_components-1,
-                                                          field_name));
-#endif
               }
-          }
+	   }
```

Comment:
There is still a space missing.

To do this automatically, you might want to run `make indent` in your build directory. It indents all source files to the required standard.